### ISSUE TITLE
NECB2017:Implementation of lighting power density upgrades

### ISSUE THREAD

**Issue by khaddad** [canmet-energy/btap_tasks#151]
### Description
Table 4.2.1.6 of the NECB 2017 lists lighting power density requirements for space types. This table contains updates to the values for LPD compared to NECB 2015. These defaults for LPDs need to be implemented. 

### Approach
After reviewing the issue, decribe your solution. If including git hyperlinks, ensure hyperlinks to existing code are to a version so it does not change with updates to git. 

### Testing Plan
Enter testing plan, if you are going to use an existing test or make a new one. 

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez
